posthog/settings/data_stores.py

```diff
@@ -100,10 +100,20 @@ def postgres_config(host: str) -> dict:
 read_host = os.getenv("POSTHOG_POSTGRES_READ_HOST")
 if read_host:
     DATABASES["replica"] = postgres_config(read_host)
+    DATABASE_ROUTERS = ["posthog.dbrouter.ReplicaRouter"]
 
 if JOB_QUEUE_GRAPHILE_URL:
     DATABASES["graphile"] = dj_database_url.config(default=JOB_QUEUE_GRAPHILE_URL, conn_max_age=600)
 
+
+# Opt-in to using the read replica
+# Models using this will likely see better query latency, and better performance.
+# Immediately reading after writing may not return consistent data if done in <100ms
+# Please edit the below and add your models
+replica_opt_in = os.environ.get("READ_REPLICA_OPT_IN", "")
+READ_REPLICA_OPT_IN: list[type] = [str_to_class(i) for i in replica_opt_in.split(",") if len(i) > 0]
```

Comment:
suggestion: Instead of `str_to_class` to get class objects here, in `db_for_read(model)`, you can check the class name of the model like so:

`model.__name__`, and check if that's in `READ_REPLICA_OPT_IN` to decide what to do.

Reply:
Oh that's nicer for sure! Thank you 🙏